SocialCareCaseViewerApi.Tests/V1/Helpers/TestHelpers.cs

```diff
@@ -143,9 +145,29 @@ public static (UpdateAllocationRequest, Worker, Worker, InfrastructurePerson, Te
                 .RuleFor(w => w.DateStart, start)
                 .RuleFor(w => w.DateEnd, end)
                 .RuleFor(w => w.IsActive, f => isActive)
+                .RuleFor(w => w.Allocations, hasAllocations ? new List<AllocationSet> { CreateAllocation(), CreateAllocation(), CreateAllocation() } : null)
+                .RuleFor(w => w.WorkerTeams, hasWorkerTeams ? new List<WorkerTeam> { CreateWorkerTeam(), CreateWorkerTeam(), CreateWorkerTeam() } : null)
                 .RuleFor(w => w.LastModifiedBy, f => createdBy ?? f.Person.Email);
         }
 
+        private static AllocationSet CreateAllocation(int? id = null, int? personId = null, int? workerId = null, int? teamId = null)
+        {
+
+            var caseStatusChoices = new List<string> { "open", "closed" };
+
+            return new Faker<AllocationSet>()
+                .RuleFor(a => a.Id, f => id ?? f.UniqueIndex + 1)
+                .RuleFor(a => a.PersonId, f => personId ?? f.UniqueIndex + 1)
+                .RuleFor(a => a.WorkerId, f => workerId ?? f.UniqueIndex + 1)
+                .RuleFor(a => a.TeamId, f => teamId ?? f.UniqueIndex + 1)
+                .RuleFor(a => a.AllocationStartDate, f => f.Date.Past())
+                .RuleFor(a => a.AllocationEndDate, f => f.Date.Past())
+                .RuleFor(a => a.CaseStatus, f => f.PickRandom(caseStatusChoices))
+                .RuleFor(a => a.CaseClosureDate, f => f.Date.Past());
```

Comment:
Can we add `.ToUniversalTime()` the date attributes? I think this caused tests to fail for Windows and MacOS before.

Reply:
Done